### DISCUSSION TITLE
Match statement cannot be used.

### DISCUSSION THREAD

**Discussion by kubotafamily** [pybricks/support#1759] [Q&A] [answered]
I want to use match statements in Pybricks Code.
Are there plans to implement this?

**Comment by dlech** [comment, marked as answer]
Pybricks is built on MicroPython, so it would have to be implemented in that project first.

Here is the feature request: https://github.com/micropython/micropython/issues/7847

But it does not sound like this is something that will be likely to ever happen.

**Reply by kubotafamily** [reply]
Thank you for your answer!
